x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/hosts/hosts_risk_column.cy.ts

```diff
@@ -7,36 +7,13 @@
 
 import { login } from '../../../tasks/login';
 import { visitWithTimeRange } from '../../../tasks/navigation';
-
 import { hostsUrl } from '../../../urls/navigation';
 import { TABLE_CELL } from '../../../screens/alerts_details';
 import { kqlSearch } from '../../../tasks/security_header';
 import { mockRiskEngineEnabled } from '../../../tasks/entity_analytics';
 
 describe('All hosts table', { tags: ['@ess'] }, () => {
-  describe('with legacy risk score', () => {
-    before(() => {
-      cy.task('esArchiverLoad', { archiveName: 'risk_hosts' });
-    });
-
-    beforeEach(() => {
-      login();
-    });
-
-    after(() => {
-      cy.task('esArchiverUnload', { archiveName: 'risk_hosts' });
-    });
-
-    it('it renders risk column', () => {
-      visitWithTimeRange(hostsUrl('allHosts'));
-      kqlSearch('host.name: "siem-kibana" {enter}');
-
-      cy.get('[data-test-subj="tableHeaderCell_node.risk_4"]').should('exist');
-      cy.get(TABLE_CELL).eq(4).should('have.text', 'Low');
-    });
-  });
-
-  describe('with new risk score', { tags: ['@serverless'] }, () => {
+  describe('with risk score', { tags: ['@serverless'] }, () => {
```

Comment:
Same about the describes :)

Reply:
[remove extra describe block](https://github.com/elastic/kibana/pull/201810/commits/6ea6296d34a6ccf1c87ee310530f41289d59be10)